nasa-gibs/worldview
# WV-3351 Integrate and validate the use of UMM-Vis to test in Worldview

## Description

Refactor layer metadata retrieval and rendering
- Update features.json to include cmrVisualizationsUrl
- Modify getVisMetadata.js to fetch metadata from cmrVisualizationsUrl if available
- Update renderSplitTitle.js and selectors.js to handle title and subtitle properties in different cases

## How To Test

- `git checkout WV-3351-umm-vis-test`
- `npm i && npm run build:verbose`
- Confirm that the app behaves normally
- You can use layer "VIIRS_SNPP_DayNightBand_AtSensor_M15" as a test case.

## PR Submission Checklist

This is simply a reminder of what we are going to look for before merging your code.

- I have read the [CONTRIBUTING](https://github.com/nasa-gibs/worldview/blob/main/.github/CONTRIBUTING.md) doc
- I have added necessary documentation (if applicable)
- I have added tests that prove my fix is effective or that my feature works (if applicable)
- Any dependent changes have been merged and published in downstream modules (if applicable)

## Merging

Please use the `squash and merge` commit method unless each commit in your branch is vital to the commit history of main.

@nasa-gibs/worldview
